src/png.imageio/png_pvt.h

```diff
@@ -4,7 +4,7 @@
 
 #pragma once
 
-#include <png.h>
+#include <libpng16/png.h>
```

Comment:
I guess this will break if/when we switch to libpng 1.8 (I'm not sure if libpng 1.7 is going to see an official release).
Should we document this somewhere?

Reply:
I'm not sure this line is critical, I was thinking at the time that it might help to guard against accidentally getting libpng15 headers (which was one battle I had at one point).

Let's keep it. I think we'll know as soon as this breaks with libpng18.